receiver/awscontainerinsightreceiver/config.go

```diff
@@ -60,4 +60,8 @@ type Config struct {
 
 	// EnableAcceleratedComputeMetrics enabled features with accelerated compute resources where metrics are scraped from vendor specific sources
 	EnableAcceleratedComputeMetrics bool `mapstructure:"accelerated_compute_metrics"`
+
+	// EnableAwsNeuronMetrics toggles Neuron monitoring where metrics are scraped from neuron monitor
+	// The default value is false.
+	EnableAwsNeuronMetrics bool `mapstructure:"neuron_metrics"`
```

Comment:
thought we will be using `EnableAcceleratedComputeMetrics` to turn off gpu monitoring all together.  is it changing with neuron?

Reply:
I dont know if we want to do that here, both are run on different instances we will basically otherwise two scrapers even for an unsupported instance, I will double check with ben though

Reply:
Yes thank you for the reminder Hyunsoo, you're exactly right, we do intend for `EnableAcceleratedComputeMetrics` to control Neuron (and EFA) metric collection as well.